### PR TITLE
Ipamhostsupport

### PR DESCRIPTION
**Pull request by akhilesh-kabade-infoblox**
Implemented gather module for ipam host, This module will gather all the hosts for the given account.

### CODE REVIEW

File: ansible_collections/infoblox/b1ddi_modules/plugins/modules/b1_ipam_host.py [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+# Copyright (c) 2021 Infoblox, Inc.
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: b1_ipam_host
+author: "Amit Mishra (@amishra), Sriram Kannan(@kannans)"
```
Comment (sriram-kannan-infoblox) [inline ansible_collections/infoblox/b1ddi_modules/plugins/modules/b1_ipam_host.py:12]: Please add ur name

Comment (akhilesh-kabade-infoblox) [reply ansible_collections/infoblox/b1ddi_modules/plugins/modules/b1_ipam_host.py:12]: Should i need to remove both names or only Amit name.

File: ansible_collections/infoblox/b1ddi_modules/plugins/modules/b1_ipam_host.py [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+    required: true
+  addresses:
+    description:
+      - Configures the name of the IP Space
+        When fetching, the address field can be in the form “a.b.c.d”. 
+    type: list
+    required: true
+  name:
+    description:
+      - Configures the name of object to fetch, add, update or remove from the system. User can also update the name as it is possible
```
Comment (sriram-kannan-infoblox) [inline ansible_collections/infoblox/b1ddi_modules/plugins/modules/b1_ipam_host.py:38]:
```suggestion
      - Configures the name of IPAM Host to fetch, add, update or remove from the system. User can also update the name as it is possible
```

File: ansible_collections/infoblox/b1ddi_modules/plugins/modules/b1_ipam_host_gather.py [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+    host: "{{ host }}"
+    api_key: "{{ api }}"
+    state: gather
+
+'''
+
+RETURN = ''' # '''
+
+def get_host(data):
+    '''Fetches the BloxOne DDI IP Space object
```
Comment (sriram-kannan-infoblox) [inline ansible_collections/infoblox/b1ddi_modules/plugins/modules/b1_ipam_host_gather.py:22]:
```suggestion
    '''Fetches the BloxOne DDI IIPAM Host object
```

**Review by sriram-kannan-infoblox — APPROVED** [review]

**Pull request merged**